concurrent/futures/thread.py

```diff
@@ -109,6 +110,7 @@ def __init__(self, max_workers=None):
         self._threads = set()
         self._shutdown = False
         self._shutdown_lock = threading.Lock()
+        self._thread_name_prefix = thread_name_prefix
```

Comment:
Ah, I missed that. It seems to be from a more recent fix (https://github.com/python/cpython/pull/2315) which should nevertheless be included here. I'll apply the second patch in another commit on this branch.